.analysis_options

```diff
@@ -1,2 +1,4 @@
 analyzer:
-  strong-mode: true
+  strong-mode:
+    implicit-casts: False
+    implicit-dynamic: False
```

Comment:
I'd rather not enable this until https://github.com/dart-lang/sdk/issues/27623 is fixed.